packages/contracts/src/RelayHub.sol

```diff
@@ -192,11 +192,29 @@ contract RelayHub is IRelayHub, Ownable, ERC165 {
     function withdraw(uint256 amount, address payable dest) public override {
         address payable account = payable(msg.sender);
         require(balances[account] >= amount, "insufficient funds");
```

Comment:
redundant check. will be done within withdrawMultiple

Reply:
Fixed.